src/main/java/htsjdk/samtools/util/IOUtil.java

```diff
@@ -87,7 +87,10 @@
     public static final long FIVE_GBS = 5 * ONE_GB;
 
     /** Possible extensions for VCF files and related formats. */
-    public static final String[] VCF_EXTENSIONS = new String[] {".vcf", ".vcf.gz", ".bcf"};
+    public static final String VCF_EXTENSION = ".vcf";
```

Comment:
Below I see fields named `XXX_FILE_EXTENSION` so the names here aren't consistent with them. I'm not sure how important consistency is in this file.